Security/src/EOMT.ps1

```diff
@@ -692,6 +716,8 @@ Please review locations and files as soon as possible and take the recommended a
     $summary = @"
 $header
 
+$Message
```

Comment:
This should be added at the end of the summary log, since it contains the next steps for the admins/users to install SU/CU